app/controllers/works_controller.rb

```diff
@@ -153,4 +152,13 @@ def editable?
   def set_presenter
     @work_presenter = WorkPresenter.new(work: @work, work_form: @work_form, version_status: @version_status)
   end
+
+  def new_work_form
+    WorkForm.new(
+      collection_druid: @collection.druid,
+      content_id: @content.id,
+      license: @collection.license,
+      access: @collection.stanford_access? ? 'stanford' : 'world'
```

Comment:
Where is this defined?

Reply:
It's a magic method that comes from the access enum.

Reply:
Should have figured it was magic.